examples/llm/aiq_baseline/langchain_corpus_ingest.py

```diff
@@ -0,0 +1,260 @@
+        } for (idx, text_line) in enumerate(emb_text_lines)]
+        """
+        data = []
+        # logger.info("Adding %s document chunks to Milvus collection %s", len(text_lines), collection_name)
+        for i, line in enumerate(tqdm(text_lines, desc="Creating embeddings")):
+            data.append({"id": i, "vector": emb_text(line).tolist()[0], "text": line})
+        """
+
+        # Insert the data into the collection
+        # FIXME: Should we add the option to update an existing collection with new documents?
+        milvus_client.insert(collection_name=collection_name, data=data)
+        logger.info(
+            "Successfully added %s document chunks to Milvus collection %s",
+            len(text_lines), collection_name)
+
+    return (milvus_client, encoder_model)
+
+
+if __name__ == "__main__":
+    import argparse
+
+    CUDA_COLLECTION_NAME = "qa_docs"
+    DEFAULT_URI = "http://localhost:19530"
+
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("--milvus_uri", "-u", default=DEFAULT_URI,
+                        help="Milvus host URI")
+    parser.add_argument("--collection_name", "-n",
+                        default=CUDA_COLLECTION_NAME,
+                        help="Collection name for the data.")
+    parser.add_argument(
+        '--dataset', type=str, default="techqa", help="Dataset folder name, "
+        "should contain corpus and train.json files."
+        "will be saved in the dataset folder")
+    parser.add_argument('--llm_generator_name', type=str,
+                        default=LLM_GENERATOR_NAME_DEFAULT,
+                        help="The LLM to use for Generation")
+    parser.add_argument('--drop_collection', action="store_true",
+                        help="Drop the collection")
+    parser.add_argument('--embedding_model', type=str,
+                        default=ENCODER_MODEL_NAME_DEFAULT,
+                        help="The embedding model")
+    parser.add_argument('--chunk_size', type=int, default=1024,
+                        help="Character chunk size when splitting the text")
+    parser.add_argument('--chunk_overlap', type=int, default=128,
+                        help="Character chunk overlap when splitting the text")
+    parser.add_argument('--metric_type', type=str, default="IP",
+                        help="Metric type. Other options are COSINE, L2")
+    parser.add_argument('--with_react_agent', action="store_true",
+                        help="Use react_agent")
+    args = parser.parse_args()
+
+    (milvus_client, encoder_model) = main(
+        milvus_uri=args.milvus_uri, collection_name=args.collection_name,
+        dataset=args.dataset, llm_generator_name=args.llm_generator_name,
+        drop_collection=args.drop_collection,
+        embedding_model=args.embedding_model, chunk_size=args.chunk_size,
+        chunk_overlap=args.chunk_overlap, metric_type=args.metric_type)
```

Comment:
It looks like you also have to pass `args.with_react_agent` based on the signature of `main`.